test/sdk/sdk.test.ts

```diff
@@ -27,14 +27,22 @@ export const getHourlyTimestamp = (hour: number) => (moment().add(hour, 'hour').
 describe('test-basic', () => {
   test('test-sdk-types', async () => {
     const turAsset = new ChainAsset({ asset: assets.tur, isNative: true });
-    const chain = new Chain({ key: 'turing-local', assets: [turAsset], defaultAsset: turAsset, endpoint: 'ws://127.0.0.1:9946', network: 'rococo', relayChain: 'local', instructionWeight: new Weight(new BN(1), new BN(2)) });
+    const chain = new Chain({ key: 'turing-local', assets: [turAsset], defaultAsset: turAsset, endpoint: 'ws://127.0.0.1:9946', relayChain: 'local', xcm: { network: 'rococo', instructionWeight: new Weight(new BN(1), new BN(2))} });
     console.log('chain: ', chain);
   });
 
   test('test-config', async () => {
     console.log('turLocal: ', chains.turingLocal);
     console.log('moonbaseLocal: ', chains.moonbaseLocal);
   });
+
+  test('test-custom-moonbase-adapter-config', async () => {
+    const { moonbaseAlpha } = chains;
+    moonbaseAlpha.xcm.instructionWeight = new Weight(new BN(1), new BN(2));
```

Comment:
Custom config

Reply:
Please add a line of comment above that code, instead adding a comment in this PR, so others can read what this line is about later.